src/common/SPELLS/deathknight.ts

```diff
@@ -301,6 +301,12 @@ const spells = spellIndexableList({
     icon: 'ability_creature_disease_02',
   },
 
+  VILE_INFUSION_BUFF: {
```

Comment:
```suggestion
  T29_VILE_INFUSION_BUFF: {
```
Marking this with T29 so its easier to find and remove later